contributing.md

```diff
@@ -0,0 +1,141 @@
+* [Definition of Ready](#definition-of-ready)
+* [Definition of Done](#definition-of-done)
+
+[How can I contribute?](#how-can-i-contribute)
+* [Reporting Bugs](#reporting-bugs)
+* [Suggesting Enhancements](#suggesting-enhancements)
+* [Your First Code Contribution](#your-first-code-contribution)
+* [Pull Requests](#pull-requests)
+
+[Conventions](#conventions)
+* [Git](#git)
+* [Commit Messages](#commit-messages)
+* [Force Pushing](#force-pushing)
+
+
+## Board and Tickets
+Here is described when tickets are ready and what stages to pass to make a ticket done.
+
+### From Backlog to Done
+
+We track the progress of the Metafacture Playground issues in the [Metafacture Fix and Playground Board](https://github.com/orgs/metafacture/projects/2). Issues move from left to right. We use the folllowing columns:
+
+*Backlog*  
+Here are all issues that are planned but not ready, have open questions and/or dependencies on other issues or on an external resource. We don't want to write down every idea about the playground so our backlog keeps manageable. We are convinced that important issues that can't be implemented now will pop up again so we don't need to write down everything that's important but not feasible now.
+
+*Ready*  
+An issue is ready if it’s possible to start working on it, i.e. there are no blocking dependencies and requirements are clear enough to start working. Dependencies are expressed through simple referencing of the blocking issue (e.g. dependes on #111), see details on referencing. Prioritized items (like bugs) are moved to the top of the ready column.
+The assignee must verify the readiness with the [Definition of Ready](#definition-of-ready).
+
+*Working*  
+When we start working on an issue, we move it to the working column. Ideally, every person should only work on one issue at a time. That way the working column provides an overview of who is currently working on what. Issues are only moved into or out of the working column by the person who is assigned. Issues in working are only reassigned by the person who is currently assigned.
+For every issue we open a feature branch that contains the corresponding issue number and additional info for convenience (using camelCaseFormatting, e.g. 111-featureDesciption).
+We include references to the corresponding issue in the commit messages.
+If the assignee thinks the issue is ready for review he/she moves it to review column and assigns a user for functional review.
+We add instructions and links for testing the changed behavior on the test system in the issue.
+
+*Review*  
+There are two kinds of Review:  
+a) Functional Review  
+If the reviewers find problems or have comments during the review, they describe the issues, providing links that show the behavior, and reassign the team member that submitted the issue, leaving the issue in the review column.
+If everything works as expected, the reviewers post a +1 comment on the issue and unassign themselves as reviewer.
+
```

Comment:
Is the +1 mandatory? Or is it sufficient to use githubs "Review changes"->"approved"?

Reply:
The functional review happens on the issue, not the PR, so there is no formal review mechanism at this point.

Reply:
> The functional review happens on the issue, not the PR,

This should be made (more) explicit in the text

Reply:
" post a +1 comment on the issue" should be clear enough. Don't know why I missed that.